usage/rest-server/src/main/java/brooklyn/rest/resources/ServerResource.java

```diff
@@ -238,6 +240,12 @@ public String getStatus() {
         return getHighAvailabilityNodeState().toString();
     }
 
+    @Override
+    public String getConfig(String configKey) {
+        ConfigKey<String> config = ConfigKeys.newStringConfigKey(configKey);
```

Comment:
Worth including an entitlements check, because the return values could contain passwords retrieved from `brooklyn.properties`. For example, something like:

    if (!Entitlements.isEntitled(mgmt().getEntitlementManager(), Entitlements.SEE_ALL_SERVER_INFO, null)) {
        throw WebResourceUtils.unauthorized("User '%s' is not authorized for this operation", Entitlements.getEntitlementContext().user());
    }